RELEASING.md

```diff
@@ -0,0 +1,144 @@
+# Releasing Kiali
+
+Kiali release a minor version every 3 weeks and eventually if required, patch releases (z-stream) are also released.
+
+The artifacts that are released are the following:
+
+* The Kiali server container image, which is published to the [Kiali Quay.io repository](https://quay.io/repository/kiali/kiali?tab=tags)
+* The Kiali operator container image, which is published to the [Kiali operator Quay.io repository](https://quay.io/repository/kiali/kiali-operator?tab=tags)
+* Helm charts for deploying the Kiali operator or the Kiali server via the Helm CLI, published to https://kiali.org/helm-charts/
+* The website that is published on https://kiali.io
+
+Tags and branches are also created in the Kiali's GitHub repositories, as a reference for rebuilds (if needed) or as a base for z-stream releases.
+
+## How releases are generated?
+
+The releases are generated by GitHub Actions pipelines. These pipelines run on every Kiali repository generating the artifacts previously mentioned:
+
+* [kiali](https://github.com/kiali/kiali/actions/workflows/release.yml)
+* [kiali-operator](https://github.com/kiali/kiali-operator/actions/workflows/release.yml)
+* [helm-charts](https://github.com/kiali/helm-charts/actions/workflows/release.yml)
+* [kiali.io](https://github.com/kiali/kiali.io/actions/workflows/release.yml)
+
+The pipelines are scheduled to run on every Friday detecting if a released is needed (on every Sprint ending a release will be generated). Also, they can be started manually filling some parameters required by the pipeline (release type, branch name and image name).
+
+## Generating a release locally (without automation)
+
+Although is not recommended, in case of any issue with GitHub Actions, following the steps that the releases pipelines are executing should be enough to generate a release.
```

Comment:
```suggestion
Although it is not recommended, in case of any issue with GitHub Actions, following the steps that the releases pipelines are executing should be enough to generate a release.
```